tests/e2e/test_requests.py

```diff
@@ -0,0 +1,195 @@
+import http
+import requests
+import numpy as np
+import os
+import pytest
+import json
+import urllib.parse
+from utils.cloud import generate_container_signature
+
+from requests_toolbelt.multipart import decoder
+
+STORAGE_ACCOUNT_NAME = os.getenv("STORAGE_ACCOUNT_NAME")
+STORAGE_ACCOUNT_KEY = os.getenv("STORAGE_ACCOUNT_KEY")
+ENDPOINT = os.getenv("ENDPOINT").rstrip("/")
+CONTAINER = "testdata"
+VDS = "wellknown/well_known_default"
+
+
+@pytest.mark.parametrize("method", [
+    ("get"),
+    ("post")
+])
+def test_slice(method):
+    meta, slice = request_slice(method, 5, 'inline')
+    expected = np.array([[116, 117, 118, 119],
```

Comment:
These tests have 3 components, 1) defining some expected value 2) calling the functions we are testing 3) asserting 1 and 2.

Can we reflow the test in that way? Now it's all a mixed together